bitcoin-explained/open-timestamps-leveraging-bitcoin-s-security-for-all-data.md

```diff
@@ -637,507 +619,455 @@ But you do see an email, so you might rightfully ask, okay, maybe this email is
+
+I think one of the nice things about Open Timestamps is that it actually resembles the idea that is mentioned in the Bitcoin white paper, [Scott Stornetta and Stuart Haber](https://www.forbes.com/sites/vipinbharathan/2020/06/01/the-blockchain-was-born-20-years-before-bitcoin/?sh=1a547035d71e), if I'm recalling these names correctly.
+They had their own idea for timestamping back in the the late 80s or the early 90s.
 And back then, Stornetta's idea, his concern was that, as long as we have paper documents, then forging paper documents, is at least kind of difficult.
 It's at least sort of like it leaves traces or you can sort of tell that something has been doctored with or messed with, or it's at least sort of possible while if we're moving.
 
-Speaker 1: 00:28:11
+Sjors Provoost: 00:28:11
 
 And you know how old paper is.
 
-Speaker 0: 00:28:13
+Aaron van Wirdum: 00:28:13
 
 Yeah, you can even tell how old paper is in many cases, or at least roughly.
 And if we're moving to a digital world, that kind of stuff becomes completely impossible and every document can be altered in ways that are just unseeable.
-So He wanted to solve that problem and he and Heber, they tried to solve this problem.
+He and Haber tried to solve this problem.
 They couldn't figure out how to solve this problem because every solution they come up with relied on some kind of third party to vouch that a document existed at a certain point in time.
 And maybe that third party would be checked by another, by fourth party, but there was always some trust involved, and that's what I want to get rid of.
 
-Speaker 1: 00:28:59
+Sjors Provoost: 00:28:59
 
-Right, and That sounds pretty problematic in the Stalin example you just mentioned.
+That sounds pretty problematic in the Stalin example you just mentioned.
 
-Speaker 0: 00:29:03
+Aaron van Wirdum: 00:29:03
 
 Exactly, yes.
 So, ultimately, they thought they were not going to be able to solve this problem, and then they wanted to at least prove that this was an unsolvable problem.
-But then while trying to prove that it was unsolvable, they came to the realization that, wait a minute, this is actually solvable if everyone in the world is party of the timestamping process.
+But then while trying to prove that it was unsolvable, they came to the realization that this is actually solvable if everyone in the world is party of the timestamping process.
 So everyone in the world submits their hashes and everyone in this world receives the combined hashes in return.
-That way, if you want to forge history, you would have to corrupt everyone or it would be obvious to everyone, like No one's going to go along with that.
+That way, if you want to forge history, you would have to corrupt everyone or it would be obvious to everyone, like no one's going to go along with that.
```

Comment:
"Merkle route" is incorrect, replace with merkle root.
Check [The style guide](https://github.com/bitcointranscripts/bitcointranscripts/blob/master/STYLE.md) for reference